src/cellEditors/CellEditor.js

```diff
@@ -242,8 +233,6 @@ var CellEditor = Base.extend('CellEditor', {
             this.grid.cellEditor = null;
             this.el.remove();
         } else if (feedback >= 0) { // never true when `feedback` undefined
-            var point = this.event.gridCell;
-            this.grid.selectViewportCell(point.x, point.y - this.grid.getHeaderRowCount());
```

Comment:
not sure why i removed this.... the header row count was probably out of date. i'll revisit.